modules/webbrowser/src/processors/webbrowserprocessor.cpp

```diff
@@ -267,6 +271,18 @@ void WebBrowserProcessor::process() {
         browser_->GetMainFrame()->ExecuteJavaScript(js_.get(), "", 1);
     }
 
+    if (dataFrames_.isReady()) {
+        auto frame = browser_->GetMainFrame();
+        size_t dataFrameNr = 0;
+        for (const auto& dataFrame : dataFrames_) {
+            nlohmann::json jsonString = *dataFrame;
```

Comment:
Is dataFrame a port or the actual data?
should it not be dataFrame->getData()

Reply:
Also, then possible to check dataFrame->isModified() or similar?

Reply:
```const shared_ptr<const inviwo::DataFrame> &dataFrame```
A pointer to the actual dataframe, so it's not posible to check isModified(). I can run:
```dataFrame->getNumberOfColumns()```
etc and it gives me what seems to be correct results for the dataframe.

Reply:
@ResearchDaniel `dataFrames_` is the port, but there is an iteratable which returns the data directly.

Alternatively, there is `auto changed = inport_.getChangedOutports();` which only returns updated ports.